cluster/gce/util.sh

```diff
@@ -959,15 +959,23 @@ function kube-down {
 # Gets the instance template for the managed instance group with the provided
 # project, zone, and group name. It echos the template name so that the function
 # output can be used.
+# Assumed vars:
+#   NODE_INSTANCE_PREFIX
 #
 # $1: project
 # $2: zone
-# $3: managed instance group name
 function get-template {
   # url is set to https://www.googleapis.com/compute/v1/projects/$1/global/instanceTemplates/<template>
-  local url=$(gcloud compute instance-groups managed describe --project="${1}" --zone="${2}" "${3}" | grep instanceTemplate)
+  local url=$(gcloud compute instance-groups managed describe --project="${1}" --zone="${2}" "${NODE_INSTANCE_PREFIX}"-group | grep instanceTemplate)
```

Comment:
I'm confused. Why don't we just change this to `gcloud compute instance-templates describe "${NODE_INSTANCE_PREFIX}"-template` and skip all this mess, rather than flakily probing the IG at all?